test/WorkItemBuiltins/get_global_id-with-global-offset.cl

```diff
@@ -0,0 +1,31 @@
+// RUN: clspv -global-offset %s -o %t.spv
```

Comment:
Can you please add a version of this test for:
```
out[0] = get_global_id(3);
```

Reply:
Done.